lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -118,17 +118,17 @@ def name
     "Core"
   end
 
-  def cmd_sess_help
-    print_line('Usage: sess <session id>')
+  def cmd_sessions_help
+    print_line('Usage: sessions <id>')
     print_line
     print_line('Interact with a different session Id.')
     print_line('This works the same as calling this from the MSF shell: sessions -i <session id>')
     print_line
   end
 
-  def cmd_sess(*args)
+  def cmd_sessions(*args)
```

Comment:
This is the best thing for my muscle memory. I'll be using this like a fiend within Meterpreter.